README_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# [Repo Name]
+
+## Contributing
+
+See [CONTRIBUTING](CONTRIBUTING.md) for additional information.
+
+### Branch flow
+
+- Main branch: `master`
+- Development branch: `develop`
+
+## Licensing
```

Comment:
We don't have to have perfect hierarchy -- `## Licensing` can go, and it can either skip right to `### Public domain`. Or we can bump `Public domain` up to an h2. Either way seems better to me.